src/app/wizard/step/cluster/template.html

```diff
@@ -52,13 +52,25 @@
         <mat-card-title>Network Configuration</mat-card-title>
       </mat-card-header>
       <div fxLayout="column">
+        <mat-form-field class="km-dropdown-with-suffix">
+          <mat-label>CNI Plugin</mat-label>
+          <mat-select [formControlName]="Controls.CNIPlugin"
+                      class="km-select-ellipsis"
+                      disableOptionCentering>
+            <mat-option [value]="cniPlugin.canal">canal</mat-option>
```

Comment:
On official pages both Canal and Cilium are starting with upper letters. Can we follow that in the UI?

Reply:
yes, I think we can display the proper names but then need to pass lowercase to the backend. I just thought that is our convention, since we show `ipvs` despite the fact [it's all caps](https://docs.projectcalico.org/networking/use-ipvs) elsewhere

Reply:
No, the IPVS was not intentional I think. Let's use official names.